crates/ruff_linter/resources/test/fixtures/refurb/FURB157.py

```diff
@@ -42,3 +42,15 @@
 Decimal("_") # Ok
 Decimal(" ") # Ok
 Decimal("10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000") # Ok
+
+# Non-finite variants
+# https://github.com/astral-sh/ruff/issues/14587
+Decimal(float(" nan "))          # Decimal(" nan ") 
+Decimal(float(" +nan "))         # Decimal(" +nan ")
+Decimal(float(" -nan "))         # Decimal(" nan "), notice the difference!
```

Comment:
I would prefer an explanatory comment here rather than a riddle :laughing: 

I suspect that the importance is the missing `-` sign but the existing comment leaves me wondering if there's more to it.